editor/plugins/editor_preview_plugins.cpp

```diff
@@ -342,6 +342,11 @@ Ref<Texture2D> EditorPackedScenePreviewPlugin::generate_from_path(const String &
 
 	Node *p_scene = pack->instantiate(); // The instantiated preview scene
 
+	if (!p_scene) {
+		print_error(vformat("Failed to generate scene thumbnail for %s : Failed to instantiate scene", p_path));
+		return Ref<Texture2D>();
```

Comment:
Is there any reason why it's `if` with `print_error` and `return`, and not a single line `ERR_FAIL_COND_V_MSG`?

Reply:
I follow the pattern from other `if` conditions above these changes. But it is a good idea to change them all to `ERR_FAIL_COND_V_MSG`.